pandasai/llm/google_palm.py

```diff
@@ -0,0 +1,85 @@
+"""Google Palm LLM"""
+from typing import Optional
+
+from google import generativeai
+
+from ..exceptions import APIKeyNotFoundError
+from .base import LLM
+
+
+class GooglePalm(LLM):
+    """Google Palm LLM"""
+
+    model: str = "models/text-bison-001"
```

Comment:
Can we make it so we support multiple models (in a similar way we did with OpenAI)

Reply:
@gventuri I have refactored it. Please check again.